test/e2e/storage/testsuites/multivolume.go

```diff
@@ -711,16 +699,30 @@ func TestConcurrentAccessToRelatedVolumes(f *framework.Framework, cs clientset.I
 		e2epod.SetAffinity(&node, actualNodeName)
 	}
 
-	// Check that all pods the same content
-	for i, pod := range pods {
-		fileName := "/mnt/volume1/index.html"
-		index := i + 1
 
-		ginkgo.By(fmt.Sprintf("Checking if the volume in pod%d has expected initial content", index))
-		commands := e2evolume.GenerateReadFileCmd(fileName)
-		_, err := framework.LookForStringInPodExec(pod.Namespace, pod.Name, commands, expectedContent, time.Minute)
-		framework.ExpectNoError(err, "failed: finding the contents of the mounted file %s.", fileName)
+	for _, pvc := range pvcs {
```

Comment:
nit: probably the if statement should only wrap the parts that are needed because a lot of the code is the same e.g.

```
for _, pvc := range pvcs {
	for i, pod := range pods {
		var fileName string
		index := i + 1
        if *pvc.Spec.VolumeMode == v1.PersistentVolumeBlock {
			// ..
        } else {
        	// ..
        }
	}
}
```

Reply:
Definitely agree! I'll refactor this out.

Reply:
Definitely agree! I'll refactor this out.